tex/appendix/ans-fo-semantics.tex

```diff
@@ -179,6 +179,24 @@ \subsection*{9.7.6}
     \emptyset$ is valid in our model.
 \end{enumerate}
 
+\subsection*{9.7.10}
+
+\begin{enumerate}
+    \item[i] We will prove $\mathcal{M} \vDash \exists x \exists y (P(x) \wedge P(y) \wedge x \neq y)$ iff $P^{\mathcal{M}}$ has at least two elements. \\\\
+    Left-to right: We will prove this using proof by contradiction. Let $\mathcal{M}$ and  $\alpha$ be arbitrary such that $\mathcal{M}, \alpha \vDash \exists x \exists y (P(x) \wedge P(y) \wedge x \neq y)$. Assume $P^\mathcal{M}$ has at most 1 element. We can distinguish two cases: $P^\mathcal{M}$ has 0 elements (1) and $P^\mathcal{M}$ has 1 element (2). \\ 
+    (1) We assume $P^\mathcal{M} = \emptyset$. Then there is no $d \in D^\mathcal{M}$ such that $P(d)$ is true. This means that the conjunction $P(x) \wedge P(y) \wedge x \neq y$ can never be true. But we assumed $\mathcal{M}, \alpha \vDash \exists x \exists y (P(x) \wedge P(y) \wedge x \neq y)$. Thus, we have a contradiction. It follows that if $P^\mathcal{M} = \emptyset$, $\mathcal{M}, \alpha \nvDash \exists x \exists y (P(x) \wedge P(y) \wedge x \neq y)$. \\
+    (2) We assume $P^\mathcal{M}$ has 1 element. Let this be an arbitrary element $d \in D^\mathcal{M}$. In order for $\mathcal{M}, \alpha \vDash \exists x \exists y (P(x) \wedge P(y) \wedge x \neq y)$ to hold, it must be the case that $\mathcal{M}, \alpha_{[x \mapsto d, y \mapsto d]} \vDash (P(d) \wedge P(d) \wedge d \neq d)$. Clearly, $P(d) \wedge P(d)$ holds, but we know $d = d$. Thus, we have a contradiction. It follows that if $P^\mathcal{M}$ has one element, $\mathcal{M}, \alpha \nvDash \exists x \exists y (P(x) \wedge P(y) \wedge x \neq y)$. \\\\
```

Comment:
Here you could argue a bit more explicitly:
Since $\mathcal{M},\alpha\vDash \exists x\exists y(P(x)\land P(y)\land x\neq y)$ we have to have two objects $d,d'\in D^\mathcal{M}$ such that $\mathcal{M},\alpha[x\mapsto d, y\mapsto d']\vDash P(x)\land P(y)\land x\neq y$. It then follows that $\alpha[x \mapsto d, y\mapsto d'](x)=d\in P^\mathcal{M}$, $\alpha[x \mapsto d, y\mapsto d'](y)=d'\in P^\mathcal{M}$, and $\alpha[x \mapsto d, y\mapsto d'](x)=d \neq d' =\alpha[x \mapsto d, y\mapsto d'](y)$. But that means that d and d' are two distinct elements of $P^\mathcal{M}$ in contradiction to the assumption.